File: Big3/CSharp/MyApp/hello.cs
using System;

namespace MyPractice
{
    public static class Hello {
       public static void greetUser()
        {
            Console.Write("Please enter your Name: ");
            string? name = Console.ReadLine();
            Console.WriteLine($"Hello {name}, its good to have you here");
            Console.WriteLine($"{name} please enter your surname: ");
            string? surname = Console.ReadLine();
            Console.Write($"Nice surname Mr {surname} ");
        } 
    }
}

File: Big3/Python/Prac/hello.py
def greetUser():
    name = input("Please enter your name: ")
    surname = input("Please enter your surname: ")
    age = input("Please enter your age: ")
    return name, surname, age   # return values

def getUserData():
    name, surname, age = greetUser()  # unpack returned values
    print(f"Hello {name} {surname}, we are aware that you are {age} years old")